doc/whats_new/v0.20.rst

```diff
@@ -39,6 +39,13 @@ Changelog
   function to return 0 when two all-zero vectors are compared.
   :issue:`12685` by :user:`Thomas Fan <thomasjpfan>`.
 
+:mod:`sklearn.utils`
+....................
+
+- |Fix| Calling :func:`utils.check_array` on `pandas.Series` with categorical
+  data, which raised an error in 0.20.0, now returns the expected output again.
```

Comment:
``categorical data`` -> something like ``pandas custom dtype``? though I'm not able to come up with another application scenario.

Reply:
Sorry, I seem to have missed this comment.

Reply:
I think it's not important :)